execute/executetest/source.go

```diff
@@ -93,22 +99,28 @@ func (src *FromProcedureSpec) Run(ctx context.Context) {
 
 	for _, t := range src.ts {
 		var max execute.Time
-		for _, tbl := range buffers {
-			t.Process(id, tbl.Copy())
+		for i, tbl := range buffers {
```

Comment:
Rather than complicate this logic, would it make more sense to create a new kind of source in `executetest` and embed `executetest.FromProcedureSpec` inside it?

Reply:
For sure, that seems the way to go.

Reply:
Done.